test/relationships.js

```diff
@@ -145,7 +148,8 @@ function getModules() {
             label: 'Rel page',
             type: 'relationship',
             withType: 'default-page',
-            withRelationships: [ '_articles._topics' ],
+            // Intentionally revisits the articles in order to test reverse relationships
```

Comment:
Normally it would be silly to loop back like this but it's a simple way to test the reverse relationship.